src/main/java/com/niyati/designpattern/behavioral/chainOfResponsibility/SubtractNumbers.java

```diff
@@ -0,0 +1,19 @@
+package com.niyati.designpattern.behavioral.chainOfResponsibility;
+
+public class SubtractNumbers implements Chain{
+    private Chain nextChain;
+
+    public void setNextChain(Chain nextChain){
+        this.nextChain=nextChain;
+    }
+
+    @Override
+    public void calculate(Numbers request) {
+        if(request.getCalculationWanted()=="subtract"){
+            System.out.println(request.getNumber1() + " - " + request.getNumber2() + "=" + (request.getNumber1()-request.getNumber2()));
+        }
+        else {
+            nextChain.calculate(request);
+        }
+    }
+}
```

Comment:
format the entire project source code and remove system.out and replace with slf4j logging